misc/bazel/internal/zipmerge/BUILD.bazel

```diff
@@ -20,6 +20,7 @@ cc_binary(
 cc_test(
     name = "test",
     size = "small",
+    linkstatic = True,  # required to build the test in the internal repo
```

Comment:
this is only required on macOS, so I suspect something's wrong with our internal toolchain there. Will need to investigate